bears/general/AnnotationBear.py

```diff
@@ -0,0 +1,72 @@
+from coalib.parsing.StringProcessing.Core import (search_for,
+                                                  search_in_between,
+                                                  unescaped_search_in_between)
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.HiddenResult import HiddenResult
+
+
+class AnnotationBear(LocalBear):
```

Comment:
commit message, rather write `Add AnnotationBear` in the shortlog and some explanation why it's cool and useful in the body